src/runnerComponentSimple.js

```diff
@@ -0,0 +1,239 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017-2023 (original work) Open Assessment Technologies SA ;
+ */
+
+/**
+ * A component that loads and instantiate a test runner inside an element:
+ * A light-weight version of `runnerComponent.js`:
+ * - removes `handlebars` template dependency
+ * - removes `ui/component` dependency, so it's api is not inherited
+ * Logic stays the same:
+ * - provider initialization
+ * - configuration validation
+ * Use it instead of `runnerComponent.js` in projects which do not use `handlebars`
```

Comment:
See https://github.com/oat-sa/tao-deliver-fe/pull/861

I didn't want to modify `runnerComponent.js` which is used in CurrentGen (what of somebody uses api inherited from `ui/component`?..), so simply made a copy. 
It could be even moved to `tao-deliver-fe`, but left here since it manages plugin initialization lifecycle, which is still more suitable for this repo.

Reply:
Moved to https://github.com/oat-sa/tao-test-runner-fe/pull/68